write-fonts/src/tables/variations/ivs_builder.rs

```diff
@@ -942,4 +1037,40 @@ mod tests {
             ],
         );
     }
+
+    #[test]
+    fn unoptimized_version() {
+        let r0 = VariationRegion::new(vec![reg_coords(0.0, 0.5, 1.0)]);
+        let r1 = VariationRegion::new(vec![reg_coords(0.5, 1.0, 1.0)]);
+
+        let mut builder = VariationStoreBuilder::unoptimized();
+        builder.add_deltas(vec![(r0.clone(), 1), (r1.clone(), 2)]);
+        // 256 won't fit in a u8 thus we expect the deltas for the column corresponding
+        // to r1 will be packed as u16
+        builder.add_deltas(vec![(r0.clone(), 1), (r1.clone(), 2)]);
+        builder.add_deltas(vec![(r0.clone(), 3), (r1.clone(), 256)]);
+        builder.add_deltas(vec![(r0.clone(), 3), (r1.clone(), 256)]);
+
+        let (ivs, _) = builder.build();
```

Comment:
maybe the test could check that the returned varidx_map (currently unused `_`) contains an identity mapping from temp ids to final varindices?